.github/workflows/publish.yml

```diff
@@ -39,15 +38,45 @@ jobs:
       - name: Run Postinstall Script
         if: steps.cache.outputs.cache-hit == 'true'
         run: npm run postinstall
+      - name: Check modified files
```

Comment:
Suggestion for name: "Check if core has changes" or "Check if core has been modified".